### ISSUE TITLE
smallvec macro in no_std code

### ISSUE THREAD

**Issue by debris** [servo/rust-smallvec#197]
hey! I'm trying to use `smallvec!` macro in `no_std` library. Is it possible?

code:

```rust
smallvec![a];
```

fails with:

```
error: cannot find macro `vec` in this scope
  --> example.rs:2:1
   |
2  | smallvec![a]
```

I already imported `extern crate alloc`

**Comment by pshaughn** [comment]
I haven't actually done this so forgive me if it's not on the right track: It looks like you need to macro_use extern crate std, since that's where vec! is defined.

**Comment by jdm** [comment]
@debris According to https://stackoverflow.com/questions/51516773/how-to-use-vec-macro-in-a-no-std-library you need to add a `#[macro_use]` to your `extern crate alloc`.

**Comment by mbrubeck** [comment]
Submitted a fix in #198.

**Comment by mbrubeck** [comment]
Version 1.2.0 has been released with the fix included.
